lib/logstash/outputs/graphtastic.rb

```diff
@@ -106,25 +106,25 @@ def receive(event)
     @retry = 1
     @logger.debug("Event found for GraphTastic!", :tags => @tags, :event => event)
     @metrics.each do |name, metric|
-      postMetric(event.sprintf(name),event.sprintf(metric),(event["@timestamp"]*1000))# unix_timestamp is what I need in seconds - multiply by 1000 to make milliseconds.
+      postMetric(event.sprintf(name), event.sprintf(metric), (event.timestamp.to_i * 1000)) #unix_timestamp is what I need in seconds - multiply by 1000 to make milliseconds.
     end
   end
-  
+
   def postMetric(name, metric, timestamp)
-    message = name+","+metric+","+timestamp.to_s
+    message = name + "," + metric + "," + timestamp.to_s
```

Comment:
I'd suggest changing this to string interpolation, as it is known to be quite faster (generates less objects)